src/DynamicRupture/FrictionLaws/FastVelocityWeakeningLaw.cpp

```diff
@@ -45,15 +45,56 @@ real FastVelocityWeakeningLaw::updateStateVariable(unsigned int pointIndex,
 
 real FastVelocityWeakeningLaw::updateMu(unsigned int ltsFace,
```

Comment:
real here but else:
```
    double muF[misc::numPaddedPoints], dMuF[misc::numPaddedPoints];
    double nr[misc::numPaddedPoints], dNr[misc::numPaddedPoints];
```

Reply:
Yes, I did not have problems with the fast velocity weakening law, when everything was in single precision.